make fails (cstring)

```
examples/bench/bench.cpp: In function ‘int whisper_bench_full(const whisper_params&)’:
examples/bench/bench.cpp:79:5: error: ‘memset’ was not declared in this scope
   79 |     memset(tokens, 0, sizeof(tokens));
      |     ^~~~~~
examples/bench/bench.cpp:6:1: note: ‘memset’ is defined in header ‘<cstring>’; did you forget to ‘#include <cstring>’?
    5 | #include <thread>
  +++ |+#include <cstring>
    6 | 
make: *** [Makefile:351: bench] Fehler 1
```

--- a/examples/bench/bench.cpp
+++ b/examples/bench/bench.cpp
@@ -1,6 +1,7 @@
 #include "whisper.h"
 
 #include <cstdio>
+#include <cstring>
 #include <string>
 #include <thread>
 

